src/fsharp/tast.fs

```diff
@@ -35,12 +35,12 @@ open Microsoft.FSharp.Core.CompilerServices
 /// Unique name generator for stamps attached to lambdas and object expressions
 type Unique = int64
 //++GLOBAL MUTABLE STATE
```

Comment:
i think now are thread safe, should i remove the `//++GLOBAL MUTABLE STATE` too? /cc @dsyme